newlib/libc/machine/microblaze/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Copyright © 2022 Keith Packard
```

Comment:
This should carry just your copyright (and your employer if necessary)

Reply:
Done.